doc/sources/guide/packaging-osx.rst

```diff
@@ -108,54 +108,65 @@ Complete guide
     $ pip install git+https://github.com/pyinstaller/pyinstaller.git@develop
 
 
+#. Export the ``HOOKSPATH`` environment variable::
+
+    $ export HOOKSPATH=`python -c "
+    import imp, os
+    print(os.path.join(imp.find_module('kivy')[1], 'tools/packaging/pyinstaller_hooks'))"`
+
 #. Package your app using the path to your main.py::
 
-    $ pyinstaller -y --clean --windowed --name touchtracer /usr/local/share/kivy-examples/demo/touchtracer/main.py
+    $ pyinstaller -y --clean --windowed --name touchtracer \
+      --additional-hooks-dir $HOOKSPATH \
+      --runtime-hook $HOOKSPATH/rt-hook-kivy.py \
```

Comment:
- It's easier to document in as the user can just copy paste the command instead of having to add the lines.
- Less prone to errors on the users side.
- Easier scriptable

Also if he needs to add more hooks he could do it on the same command line. Otherwise he would have to write code in the spec file which extends the list which is returned by ``get_hooks()``